### DISCUSSION TITLE
problems creating a jar

### DISCUSSION THREAD

**Discussion by starghoost** [jagrosh/MusicBot#1424] [Troubleshooting]
I try to create a Jar but I have these errors

![image](https://github.com/jagrosh/MusicBot/assets/60016557/715005a2-d5ad-48a7-9e1e-59034e5fb324)

![image](https://github.com/jagrosh/MusicBot/assets/60016557/6aedd77e-42f0-40ab-beb3-e12e24f2622e)

![image](https://github.com/jagrosh/MusicBot/assets/60016557/d9271477-12bd-4d33-abe1-abf31161dea0)

**Comment by jagrosh** [comment]
Building the jar yourself is not supported.
